### DISCUSSION TITLE
How to prune docker on the camera?

### DISCUSSION THREAD

**Discussion by igor-rud-kr** [AxisCommunications/acap-computer-vision-sdk-examples#64] [Q&A] [answered]
After I converted docker to SSL, I am getting the following error:

docker -H tcp://$AXIS_TARGET_IP:$DOCKER_PORT system prune -af

Error response from daemon: Client sent an HTTP request to an HTTPS server.

**Comment by Corallo** [comment, marked as answer]
Hi @igor-rud-kr 
Which docker port did you use?

**Reply by igor-rud-kr** [reply]
I used 2376

**Reply by Corallo** [reply]
2376 is for the secure connection that rely on TLS.
Check if your docker ACAP has TLS on (you can check in plain config in the camera gui).
If you want to use TLS you need to provide the flag --tlsverify to your docker command and also the certificate files.
If you don't want to use TLS you should use the 2375 port, and disable TLS from the docker ACAP

**Reply by igor-rud-kr** [reply]
thank you, this did work
